pillar/lax-public.sls

```diff
@@ -70,6 +70,12 @@ elife:
                 service_template: bot-lax-adaptor-service
                 num_processes: 1
 
+    glencoe:
+        skip_requests_cache_by_env: 
```

Comment:
just needs to be a single boolean value that changes by environment? :thinking: 
Like e.g. `pillar.journal.submit_url` as an example?

Reply:
I'll take a second look

Reply:
> just needs to be a single boolean value that changes by environment? 

I've taken a look and what you're suggesting and I think my approach is better. In this case we have one file to change where a value is set. In your approach (correct me if I'm wrong) there are N files to modify, one for each environment. That would seem to encourage file sprawl and obfuscation at the expense of simpler formulas (I think? is the intent to keep environment wrangling out of applications?). Looking at some of the `environment-whatever-public.sls` files, it doesn't appear to have stopped the use of value templating or using environment names in pillar values.

I'm not on a crusade, I think we're just merrily dolling out all the rope we need to hang ourselves with. I can understand partitioning public/private pillar data, and perhaps that was the top of the slippery slope, but not further partitioning by environment (and do we have individual project instances in there? `environment-title-length-...` ?). We should have found some way to push per-environment stuff back into the public/private project pillar data. Maybe this: https://docs.saltstack.com/en/latest/ref/states/top.html#multiple-environments . I never had the time to bend my head around it though.

I'll change things so this pillar value is changed via the environment files